Add CHT-Sync and CHT-Pipeline Documentation (#911)
medic/cht-docs: content/en/core/overview/data-flows-for-analytics.md

Data in the views and functions mentioned in this section is as accurate as the accuracy of the SQL queries. Best practice is to begin the process of defining these objects at design in order to align analytics and dashboards requirements with workflows being deployed.

#### CHT-Sync

> We have built [cht-sync](https://github.com/medic/cht-sync) to be able to sync data from CouchDB to PostgreSQL. It uses logstash and PostgREST to replicate data from CouchDB to PostgreSQL in a more real-time manner. It is a work in progress and is not yet ready for production use. It is currently being used in a few deployments to test its performance and reliability.

Reviewer: Try to avoid the "we" word as CHT tools are community built and supported. Try something like "cht-sync is able to...".

Also, this is a production ready release so let's drop the beta warning. We _want_ people to use it in prod and will support and fix any bugs they may find. Honestly, I feel more confident about cht-sync than cht-couch2pg at this point!

```suggestion
[cht-sync](https://github.com/medic/cht-sync) is able to sync data from CouchDB to PostgreSQL. It uses logstash and PostgREST to replicate data from CouchDB to PostgreSQL in a real-time manner.
```

#### CHT-Pipeline

> We have built [cht-pipeline](https://github.com/medic/cht-pipeline) to help witht the transformation of the data we get from the CHT to a format that is more suitable for analytics. We make use of [DBT](https://www.getdbt.com/) to define the models that are tanslated into Postgres tables or views. It then becomes easier to query the data in the analytics platform of choice, we recommend [Superset](https://superset.apache.org/). See these instructions on how to set up and use cht-pipeline.

Reviewer: Same as above...

```suggestion
[cht-pipeline](https://github.com/medic/cht-pipeline) helps with the transformation of the data from the CHT to a format that is more suitable for analytics. It uses [DBT](https://www.getdbt.com/) to define the models that are translated into Postgres tables or views. It then becomes easier to query the data in the analytics platform of choice, for example [Superset](https://superset.apache.org/). See these instructions on how to set up and use cht-pipeline.
```

Reviewer: > See these instructions on how to set up and use cht-pipeline.

Are you missing a link there?
